docs/notebooks/rembo.pct.py

```diff
@@ -0,0 +1,246 @@
+# %% [markdown]
+# # Noise-free optimization on a toy high-dimensional problem using Random EMbedding Bayesian Optimization (REMBO).
+# This notebook demonstrates a simple method for optimizing a high-dimensional (100-D) problem, where standard BO methods have trouble.
+
+# %%
+
+import math
+import gpflow
+import numpy as np
+import tensorflow as tf
+import tensorflow_probability as tfp
+
+np.random.seed(1793)
+tf.random.set_seed(1793)
+
+# %% [markdown]
+# ## Describe the problem
+#
+# In this example, we augment the standard two-dimensional Michalewicz function with 98 dummy dimensions to obtain a 100-dimensional problem over the hypercube $[0, \pi]^{100}$.
+#
+# We compare three approaches to optimizing this problem. The first uses a standard GP model over all 100 dimensions, using expected improvement as our acquisition function. We compare this to two Random EMbedding Bayesian Optimization (REMBO; see <cite data-cite="wang2013bayesian"/>) approaches.
```

Comment:
it would be good to add a brief explanation why this approach will not perform well